shell/browser/native_window_views.cc

```diff
@@ -1062,6 +1066,18 @@ void NativeWindowViews::SetIgnoreMouseEvents(bool ignore, bool forward) {
           .source_bitmap = x11::Pixmap::None,
       });
     }
+  } else {
+    // note: this assumes we are in wayland, but Ozone supports X11 too
+    // is there a way to confirm we are indeed on the wayland ozone platform?
```

Comment:
@loc found that we can use https://source.chromium.org/chromium/chromium/src/+/main:ui/base/linux/linux_ui_delegate.h;l=16-19;drc=7776bbb38c4e394b5be085bc8c5bc02df5fa22dc — i'll add that shortly.